File: fix_syntax.cs
using System;
using System.IO;

class Program
{
    static void Main()
    {
        string filePath = @"c:\Users\hp\OneDrive\Desktop\Raptorflow\src\pages\app\MuseChatPage.jsx";
        string content = File.ReadAllText(filePath);
        
        // Replace the problematic line
        content = content.Replace("                  )})", "                  ))");
        
        File.WriteAllText(filePath, content);
        
        Console.WriteLine("Syntax error fixed");
    }
}

File: fix_syntax.js
import fs from 'fs';

const filePath = 'c:\\Users\\hp\\OneDrive\\Desktop\\Raptorflow\\src\\pages\\app\\MuseChatPage.jsx';

let content = fs.readFileSync(filePath, 'utf8');

// Replace the problematic line
content = content.replace(/ {18}\)}\)/g, '                  ))');

fs.writeFileSync(filePath, content);

console.log('Syntax error fixed');
